lieu-vite-plugin.ts

```diff
@@ -0,0 +1,42 @@
+import {Plugin} from 'vite';
+
+export default function lieuPlugin(): Plugin {
+    return {
+        name: 'vite:lieu',
+
+        handleHotUpdate(ctx) {
+            console.log(ctx);
+        },
+
+        async resolveId(id) {
+            return null;
+        },
+
+        load(id, opt) {
+            return null;
+        },
+
+        transform(code, id, opt) {
+            if (!id.endsWith('.lieu')) {
+                return null;
+            }
+
+            code = code.replace(/<script>([\s\S]*?)<\/script>/g, (_, group) => group);
```

Comment:
Extracts script from within `<script>` tags
Then extract styles from `<style>` tags and transform to definition.styles formatting.
Then add import of the css tagged template function.
Then remove `<style>` block from file.